docs/docs/recipes.md

```diff
@@ -131,6 +131,54 @@ Showtime.
 
 ## Querying data
 
+### Using PageQuery
+
+You can use the `graphql`-tag to query data in your pages.
+
+#### Directions
+
+1. Import `graphql` from `gatsby`.
+
+2. Export a constant named `query` and set its value to be a `graphql` [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) with the query between two backticks.
+
+3. Destruct `data` from props.
+
+4. The `data` variable now holds the queried data with the expected shape.
```

Comment:
If you end up going with the simplified terminology suggestion above, this could flow a little better:
```suggestion
4. The `data` variable which holds the queried data with the expected shape can be referenced in JSX to output HTML.
```